src/bench/orphan_tx_pool.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <bench/bench.h>
+
+#include <amount.h>
+#include <net.h>
+#include <net_processing.h>
+#include <primitives/transaction.h>
+#include <random.h>
+#include <script/script.h>
+#include <sync.h>
+
+#include <vector>
+
+static constexpr CAmount CENT{1000000};
```

Comment:
Why is this needed? values should not matter for performance

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/19377#issuecomment-650013735).